resources/assets/js/components/stekpakket/StekPakket.vue

```diff
@@ -0,0 +1,387 @@
+<template>
+	<div class="container stekpakket">
+		<div class="row align-items-center">
+			<div class="col-lg-2"></div>
+			<div class="col-auto">
+				<img src="/dist/images/beeldmerk.png" alt="C.S.R. Delft" class="beeldmerk">
+			</div>
+			<div class="col">
+				<h1>De prijs van de stek</h1>
+				<div class="subtitel">Stel uw stekpakket samen</div>
+			</div>
+			<div class="col-lg-2"></div>
+		</div>
+		<div class="row align-items-center">
+			<div class="col-lg-2"></div>
+			<div class="col-lg">
+				<p>
+					Hieronder kunt u uw eigen stekpakket samenstellen.
+					U selecteert eerst een basispakket en past dit vervolgens aan naar uw smaak.
+					De kosten van uw stekpakket worden maandelijks afgeschreven van uw CiviSaldo.
+					Het stekpakket kunt u per maand bijstellen.
+				</p>
+			</div>
+			<div class="col-lg-2"></div>
+		</div>
+		<div class="row equal">
+			<div class="col-12"><h2>Kies een basispakket</h2></div>
+			<div class="col-xl-3 col-sm-6" v-for="pakket in $props.basispakketten">
+				<div class="pakket" :class="{ 'actief': gekozenBasispakket === pakket.titel }" @click="kiesBasispakket(pakket.titel, pakket.niveau)">
+					<div class="titel">{{ pakket.titel }}</div>
+					<div class="usp" v-for="usp in pakket.usps"><i class="fa fa-check"></i> {{ usp }}</div>
+					<div class="prijs">
+						<div class="getal">&euro; {{ pakket.euro }}<span>,{{ (pakket.centen < 10 ? '0' : '') + pakket.centen }}</span></div>
+						<div class="per">per maand</div>
+					</div>
+				</div>
+			</div>
+		</div>
+		<div class="row" v-if="gekozenBasispakket" id="configuratie">
+			<div class="col-12"><h2>Configureer uw stekpakket</h2></div>
+		</div>
+		<div class="row" v-if="gekozenBasispakket">
+			<div class="col-lg-1 col-xl-2"></div>
+			<div class="col-sm">
+				<template v-for="(groep, index) in opties">
```

Comment:
Met css `column-count: 2` kun je dit 1 for loop maken.

Reply:
Ja maar gedoe met mobiel 🤷‍♂️

Reply:
1 kolom op mobiel :P

Reply:
Bij een oneven aantal splitst `column-count` ook blokken op, of kan je dat uitzetten?

Reply:
(dit: https://www.w3schools.com/code/tryit.asp?filename=GDC7XN6T6E17)

Reply:
Weet ik niet.